### PR TITLE
Add API underpinnings for getting Image Captions.

### PR DESCRIPTION
**Pull request by dbrant**

### CODE REVIEW

File: app/src/main/java/org/wikipedia/dataclient/mwapi/media/MediaHelper.kt [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import org.wikipedia.dataclient.Service
+import org.wikipedia.dataclient.ServiceFactory
+import org.wikipedia.dataclient.WikiSite
+
+import java.util.HashMap
+
+import io.reactivex.Observable
+
+object MediaHelper {
+    private const val COMMONS_DB_NAME = "commonswiki";
```
Comment (cooltey) [inline app/src/main/java/org/wikipedia/dataclient/mwapi/media/MediaHelper.kt]: No `;` needed in a Kotlin file

File: app/src/main/java/org/wikipedia/dataclient/Service.java [hunk 2]
```diff
@@ -344,4 +348,8 @@
                                              @Nullable @Field("summary") String summary,
                                              @NonNull @Field("token") String token,
                                              @Nullable @Field("assert") String user);
+
+
+
+
```
Comment (cooltey) [inline app/src/main/java/org/wikipedia/dataclient/Service.java]: Adding multiple empty lines by accident?

File: app/src/main/java/org/wikipedia/dataclient/mwapi/media/MediaHelper.kt (hunk 1 above)
Comment (dbrant) [reply app/src/main/java/org/wikipedia/dataclient/mwapi/media/MediaHelper.kt]: done

File: app/src/main/java/org/wikipedia/dataclient/Service.java (hunk 2 above)
Comment (dbrant) [reply app/src/main/java/org/wikipedia/dataclient/Service.java]: whoops

**Pull request merged**